subsys/bluetooth/audio/csip_crypto.h

```diff
@@ -22,9 +22,10 @@
  * used in RSIs - Used by the Coordinated Set Identification service and
  * profile.
  *
- * @param sirk  16 byte LS byte first SIRK
- * @param r     3 byte LS byte first random value
- * @param out   3 byte LS byte first output buffer
+ * @param sirk  16 byte LS byte first SIRK in little endian byte order.
+ * @param r     3 byte LS byte first random value in little endian byte order.
+ * @param out   3 byte LS byte first output buffer in little endian byte order.
```

Comment:
We should consider also updating the documentation of `bt_crypto_aes_cmac` to specific that it expect the data as BE